google-cloud-core/pom.xml

```diff
@@ -87,5 +87,10 @@
       <artifactId>guava-testlib</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.9.5</version>
```

Comment:
This should have a <scope>test</scope> modifier so that it's not included in the non-test JAR. :)